README.md

```diff
@@ -16,6 +16,12 @@ ensure you have docker and docker compose installed and working.
 ```
 $ docker compose up
 ```
+
+### Regenerate Swagger docs
+```
+$ swag init --parseDependency && swag fmt
+```
```

Comment:
Do you think this would be worth while to do as part of the Dockerfile when the QSOLink container is built?  I poked around in there, and it looks like Swag doesn't install inside the container as part of the build either.